packages/liquidator/test/Liquidator.js

```diff
@@ -1659,6 +1686,464 @@ contract("Liquidator.js", function(accounts) {
             }
           );
         });
+        describe("Liquidation via DSProxy", () => {
```

Comment:
pretty in depth tests. please let me know if there is an edge case I dont cover!